docker/containers/docker_nightly/Dockerfile

```diff
@@ -0,0 +1,21 @@
+ARG SOURCE=ghcr.io/neuralmagic/cuda-python3.8
+# how were these versions selected before?
+ARG TORCH_VERSION=1.10.2
+ARG TORCHVISION_VERSION=0.11.3
+ARG CUDA=113
+
+FROM $SOURCE
+
+RUN python3.8 -m pip install --upgrade pip \
+    && python3.8 -m pip install --upgrade setuptools
+
+ARG CUDA 
+ARG TORCH_VERSION
+ARG TORCHVISION_VERSION
+
+RUN python3.8 -m pip install torch==${TORCH_VERSION}+cu${CUDA} torchvision==${TORCHVISION_VERSION}+cu${CUDA} -f https://download.pytorch.org/whl/torch_stable.html \
+    && python3.8 -m pip install --no-cache-dir "sparseml-nightly[onnxruntime,torchvision,transformers,yolov5,ultralytics]" 
```

Comment:
Are pushes to nightly blocked on previous nightly passing?

Reply:
there will be one follow-up PR such that containers aren't built if the wheels they need fail to build. So the dependency is on the wheel building.